testsuite/run_sets/build_validation_retail.yml

```diff
@@ -0,0 +1,20 @@
+# This file describes the order of features in a Build Validation test suite run.
+
+## NOTE: This stage can't be run in parallel
```

Comment:
Consider deploy and setup three branch network if that's the only way to parallelize it.
As otherwise, we will spend around more 6 hours when it can be done in 1.30h

Reply:
Yes, I need to revert this change to yaml files. I did not think about parallelization, my bad.

Reply:
remark changed.